reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,76 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.22.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+* Support for 64-bit only ARM systems.<!--AC-1890, AC-2428-->
+* Support for installing relevant 32-bit NVIDIA userspace binaries on AMD64 systems.<!--AC-2450-->
+* To allow easier identification and use of instances, you can now explicitly name an instance when creating it. See [How to create an instance](https://anbox-cloud.io/docs/howto/instance/create).<!--AC-2395-->
+* Support for Vulkan 1.3 for Android 13 images.<!--AC-2278-->
+* VHAL enhancements:
+    - Configuration returned by the external APIs have a new field containing a name and a list of area names for the standard VHAL properties.<!--AC-2290 and AC-2289-->
```

Comment:
What are "external APIs" in this context?

Reply:
On second thought, "external" is not necessary as we won't be discussing about any other API in release notes. I will remove it.